### PR TITLE
Added_state_names

### PR DESCRIPTION
**Pull request by varun-TR**
Added states names. 

Screenshot 2024-11-15 at 8 58 11 PM

### CODE REVIEW

File: swe.js [hunk 1]
```diff
@@ -80,10 +86,8 @@ function loadMap() {
             .setContent(content)
             .openOn(map);
     });
-
-    
 }
-
+// Function to copy coordinates to clipboard
```
Comment (ZihengSun) [inline swe.js:90]: this is duplicated

File: swe.js [hunk 2]
```diff
@@ -106,6 +110,209 @@ function copyCoordinates(lat, lon) {
         alert('Coordinates copied to clipboard!');
     }
 }
+function add_swe_predicted_geotiff(date){
```
Comment (ZihengSun) [inline swe.js:113]: These function are all duplicated. Please clean up your PR

**Review by ZihengSun — CHANGES_REQUESTED** [review]
There are lots of duplicated functions in there. Please clean up.

File: swe.js (hunk 1 above)
Comment (varun-TR) [reply swe.js:90]: sure will resolve it asap.

File: swe.js (hunk 2 above)
Comment (varun-TR) [reply swe.js:113]: okay professor

**Pull request closed without merging**